src/en/config-gce.md

```diff
@@ -0,0 +1,227 @@
+Title: Configuring Juju for Google Compute Engine
+
+# Configuring Juju for Google Compute Engine
+
+This process requires you to have a Google Compute Engine (GCE) account. If you
+do not yet have one, log into your Google account and sign up at:
+[cloud.google.com/compute/](https://cloud.google.com/compute/)
+
+If you have not run Juju before, you should start by generating a generic 
+configuration file for Juju, using the command:
+
+```
+juju generate-config
+```
+
+This will generate a file, `environments.yaml`, which will be put in your
+`~/.juju/` directory (and will create the directory if it doesn't already
+exist).
+
+**Note:** The above command will not overwrite your existing environments.yaml
+file, or output to stdout. In order to see the boilerplate environments.yaml on
+stdout you need to append the `--show` option. This is helpful if you have an
+existing environments.yaml and just need to add a section. E.g. 
+`juju generate-config --show`. You can then copy and paste the needed section.
+
+The generic configuration section generated for Windows Azure will look
+something like this:
+
+```
+    gce:
+      type: gce
+    
+      # Google Auth Info
+      # The GCE provider uses OAuth to authenticate. This requires that
+      # you set it up and get the relevant credentials. For more information
+      # see https://cloud.google.com/compute/docs/api/how-tos/authorization.
+      # The key information can be downloaded as a JSON file, or copied, from:
+      #   https://console.developers.google.com/project/<projet>/apiui/credential
+      # Either set the path to the downloaded JSON file here:
+      auth-file:
+    
+      # ...or set the individual fields for the credentials. Either way, all
+      # three of these are required and have specific meaning to GCE.
+      # private-key:
+      # client-email:
+      # client-id:
+    
+      # Google instance info
+      # To provision instances and perform related operations, the provider
+      # will need to know which GCE project to use and into which region to
+      # provision. While the region has a default, the project ID is
+      # required. For information on the project ID, see
+      # https://cloud.google.com/compute/docs/projects and regarding regions
+      # see https://cloud.google.com/compute/docs/zones.
+      project-id:
+      # region: us-central1
+    
+      # The GCE provider uses pre-built images when provisioning instances.
+      # You can customize the location in which to find them with the
+      # image-endpoint setting. The default value is the a location within
+      # GCE, so it will give you the best speed when bootstrapping or adding
+      # machines. For more information on the image cache see
+      # https://cloud-images.ubuntu.com/.
+      # image-endpoint: https://www.googleapis.com
+```
+
+In order to find out the necessary values to add to the configuration, you will
```

Comment:
hmmm. I like it that it explicitly says you are adding things to the configuration. How about 
"To determine the values to add..."?